### PR TITLE
Serpico Dev Build Install Script

### PR DESCRIPTION
**Pull request by r3b00t5**
Resolves 95% of the headache from trying to install manually. Needs a second look.

### CODE REVIEW

File: scripts/serpicoInstall.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# serpicoInstall.sh
+
+#!/bin/bash
+
+username="r3b00t"
```
Comment (Petessss) [inline scripts/serpicoInstall.sh]: username should not be hard coded.  It should be something like username=$(whoami)

Comment (r3b00t5) [reply scripts/serpicoInstall.sh]: Good point. Will change and re-submit.

File: scripts/serpicoInstall.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+gem install bundler
+
+# The Gemfile has an issue during installation, run the below to preemptively resolve it
+gem install msfrpc-client -v 1.0.3
+
+bundle install
+
+# Initialize the findings database
+ruby scripts/first_time.rb
```
Comment (Petessss) [inline scripts/serpicoInstall.sh]: How is input for first_time.rb handled?

Comment (r3b00t5) [reply scripts/serpicoInstall.sh]: Currently, that needs to be run manually. I haven't checked if first_time.rb has any arguments, but I assume I can prompt the user and pass them along with the read command.

**Comment by Petessss** [comment]
I'll test and review the script this week.

**Comment by r3b00t5** [comment]
I haven't had much chance to work on it these past few days. It was something I threw together to resolve some of the hurdles I ran into during installation. I appreciate the feedback and review.

**Pull request merged**